### ISSUE TITLE
unrecognized methods in standard library

### ISSUE THREAD

**Issue by heavenlyhash** [go-lang-plugin-org/go-lang-idea-plugin#1150]
Starting with 1512d1b531d8b5d46130b48eda6873bc45cb8c4c , many methods in the standard library are not recognized.

One example is in this snippet:
```
func (c *Config) Marshal() []byte {
	var buf bytes.Buffer
	if err := toml.NewEncoder(&buf).Encode(c); err != nil {
		panic(err)
	}

	return buf.Bytes()
}
```

`buf.Bytes()` is highlighted as an error, and mouseover contains two notices:  "cannot use buf.Bytes() (type unknown) as type []byte in return argument" and "Unresolved symbol: 'Bytes'"

All code inside my project/gopath appears to be able to refer amongst itself fine.

Most but not not all standard library references are broken.  I have not been able to fully determine the pattern.  This snippet: 

`fmt.Fprintf(os.Stderr, "Usage: %v [options]\n\n", os.Args[0])`

... has no trouble resolving `Fprintf`, but it gives an error highlight to `os.Stderr` and says "cannot use os.Stderr (type *File) as type unknown in argument to Fprintf".

Behavior tested and appears identical on go sdk 1.4 and 1.3.1.

**Comment by mtoader** [comment]
Thanks for your report. We are aware of this and working towards fixing it.

**Comment by dlsniper** [comment]
This should be fixed in 1.0
